documents/ComputeCpp_Overview.asciidoc

```diff
@@ -0,0 +1,65 @@
+:homepage: http://www.codeplay.com/computecpp
+:doctype: article
+:version: 0.1.1
+:author: Codeplay Software Ltd. <sycl@codeplay.com>
+:source-highlighter: coderay
+:coderay-css: style
+:copycss:
+:linkcss:
+
+ifdef::env-github[]
+:imagePackageContent: /codeplaysoftware/computecpp-sdk/wiki/images/ComputeCppPackageContent.png
+:computecppGettingStarted: https://github.com/codeplaysoftware/computecpp-sdk/wiki/ComputeCpp_GettingStarted
+endif::env-github[]
+ 
+ifndef::env-github[]
+include::adoc_links.adoc[]
+:imagePackageContent: images/ComputeCppPackageContent.svg
+:computecppGettingStarted: https://www.codeplay.com/products/computesuite/computecpp/reference/ComputeCpp_GettingStarted
+endif::env-github[]
+
+[abstract]
+==  ComputeCpp(TM) Community Edition (CE): Beta Release
+
+ComputeCpp Community Edition (CE) is a heterogeneous parallel programming
+platform that provides a beta pre-conformant implementantion of SYCL(TM) 1.2
+Khronos link:{syclSpecUrl}[specification].
+
+ComputeCpp CE {version} beta release includes the ComputeCpp CE Package version
+{version} and ComputeCpp CE SDK version {version}. The supported OpenCL 1.2
+platforms for ComputeCpp CE beta release, are AMD(TM) and Intel(TM).
+
+This ComputeCpp Package version {version} supports SYCL pass:[C++] libraries
+for better integration with pass:[C++] frameworks and applications like
+link:{syclParallelStlUrl}[SYCL Parallel STL], math libraries like link:{eigenSYCLUrl}[SYCL Eigen Tensor Library] and machine vision libraries.
+
+[ComputeCppPackage]
+.ComputeCpp Package Components
+image::{imagePackageContent}[ComputeCpp Package Components]
+
+
+The ComputeCpp Package includes the components of the ComputeCpp Community Edition
+beta implementation of SYCL 1.2. ComputeCpp Package includes the pass:[compute++] SYCL
+device compiler and the SYCL Runtime library along with its headers. The
+ComputeCpp Package can be downloaded from the
+link:{computecppUrl}[ComputeCpp Download Page], and a link:{computecppGettingStarted}[Getting Started guide] is available.
+
+Further details on how to integrate the ComputeCpp Package to an existing pass:[C++] application 
+can be found in the link:{computecppIntegrationGuide}[ComputeCpp Integration Guide].
+
+The ComputeCpp SDK complements the ComputeCpp Package with build system
+integration, sample code and documentation. The ComputeCpp Package is available at
+link:{computecppSdkUrl}[computecpp-sdk @ *GitHub*].
```

Comment:
This link is wrong, and I know that it will have been wrong before, but this is a good opportunity to fix it.